SmokeTests/spm/App/ViewController.swift

```diff
@@ -0,0 +1,38 @@
+/*
+* Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+* This product includes software developed at Datadog (https://www.datadoghq.com/).
+* Copyright 2019-Present Datadog, Inc.
+*/
+
+import UIKit
+import DatadogApollo
+
+internal class ViewController: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Test that DatadogApollo APIs are visible and can be instantiated:
+        let interceptor = DatadogApolloInterceptor()
+        let interceptorWithPayloads = DatadogApolloInterceptor(sendGraphQLPayloads: true)
+        
+        // Verify the interceptors are not nil
+        if interceptor as Any? != nil && interceptorWithPayloads as Any? != nil {
```

Comment:
Do you need `as Any?`? Or can we simply check is there are not `nil`?
```suggestion
        if interceptor != nil && interceptorWithPayloads != nil {
```

Reply:
We can actually remove the if all together, they can never be nil.